website/src/pages/Documentation/index.js

```diff
@@ -269,6 +270,20 @@ function Documentation () {
                                 </div>
                             </section>
                         </section>
+                        <section id="rounded-buttons" class="mt-7">
+                        <SubsectionHeader title="Rounded Buttons" to="rounded-buttons" />
+                        <div className="pl-8 text-lg">
+                           <p className="mb-3">
+                             To add the rounded buttons with a particular radius, you can use various classes like `rounded-x-btn` where x is the radius in pixels which you want. Here x ranges from `5`(border-radius = 5px) to `fully rounded`(border-radius = 50%).
+                             For example:
+                           </p>
+                           <Code code = {`<button class="rounded-5-btn">Rounded Button</button>`} language="html"/>
```

Comment:
the classes in the code snippet should be the same as the classes used for the code example.

Reply:
@shahednasser  Done.

Reply:
@shahednasser  Done the changes.

Reply:
@shahednasser  the changes are done